user/ip-addresses.md

```diff
@@ -0,0 +1,15 @@
+---
+title: IP Addresses
+layout: en
+permalink: /user/ip-addresses/
+---
+
+For some it might be necessary to know the public IP addresses of build machines in order to whitelist them to access some internal resources. We run builds in a variety of different infrastructures, so which one you use determines which IP range you need to whitelist.
```

Comment:
I'm not quite sure what part I should replace with that. It seems like replacing "For some it might be necessary to know" with "If you need to know", but there's no "…then xyz" to correspond with the "if", and I couldn't think of what to write for that.